### PR TITLE
Format author timestamps

### LINKED ISSUE #10: Format lastEditedAt

### PR DESCRIPTION
**Pull request by GeekPro101**
Fixes #10
image

### CODE REVIEW

**Review by apple-phi — COMMENTED** [review]
(I've just brought this PR up-to-date with the main branch)

File: src/lib/index.ts [hunk 1]
```diff
@@ -57,3 +58,11 @@ export function makeSubtitle(html: string): string | null {
 		? stripHtmlTags(firstParagraph).slice(0, 100).trim() + '...'
 		: firstParagraph;
 }
+
+const authorTimestampFormat = "HH:mm EEE d MMMM yyyy"
```
Comment (apple-phi) [inline src/lib/index.ts:62]: I'm not convinced that the exact time and day of the week is necessary

```suggestion
const authorTimestampFormat = "dd MMMM yyyy"
```

**Review by apple-phi — CHANGES_REQUESTED** [review]

File: src/lib/index.ts (hunk 1 above)
Comment (GeekPro101) [reply src/lib/index.ts:62]: Precise times can sometimes be helpful, and I don't see why we'd want to not provide info we have
Day of the week was an aesthetic thing - naked dates are a bit ugly I think

File: src/lib/index.ts [hunk 2]
```diff
@@ -59,10 +59,40 @@ export function makeSubtitle(html: string): string | null {
...
+    if (j === 2 && k !== 12) {
+        return i + "nd";
+    }
+    if (j === 3 && k !== 13) {
+        return i + "rd";
+    }
+    return i + "th";
+}
+
+function numberToOrdinal(i) {
```
Comment (apple-phi) [inline src/lib/index.ts:62]: `i` implicitly has an `any` type here, needs to be annotated. Also is it possible to make this more concise? Consider:
https://github.com/moment/luxon/issues/118#issuecomment-781192208

**Review by apple-phi — CHANGES_REQUESTED** [review]

File: src/lib/index.ts (hunk 2 above)
Comment (GeekPro101) [reply src/lib/index.ts:62]: We could make it shorter, but does it really need to be much more concise? It's fairly readable atm which I don't think is the case for the one you linked

File: src/lib/index.ts [hunk 3]
```diff
@@ -57,3 +58,25 @@ export function makeSubtitle(html: string): string | null {
...
+	}
+	if (j === 3 && k !== 13) {
+		return i + 'rd';
+	}
+	return i + 'th';
+}
+
+export function formatAuthorTimestamp(timestamp: string): string {
+	const timestampDateTime = DateTime.fromISO(timestamp, { zone: 'Europe/London' });
+	console.log(numberToOrdinal(timestampDateTime.day));
```
Comment (apple-phi) [inline src/lib/index.ts:79]: Remove this `console.log` and then we're good to go!

**Review by apple-phi — CHANGES_REQUESTED** [review]

**Review by apple-phi — APPROVED** [review]

**Pull request merged**